test/functionalTest/cases/1164_v2_update_entities_attr_empty_value/update_entities_attr_empty_value.test

```diff
@@ -41,14 +43,27 @@ orionCurl --url /v2/entities --payload "$payload" --json
 echo
 echo
 
-echo "02. PUT /v2/entities/E1/attrs/A1 (empty)"
-echo "==============================================="
+echo "02. GET /v2/entities/E1/attrs/A1"
+echo "================================"
+orionCurl --url /v2/entities/E1/attrs/A1 --json
+echo
+echo
+
+
+echo "03. PUT /v2/entities/E1/attrs/A1 (empty)"
+echo "========================================"
 payload='{ }'
 orionCurl --url /v2/entities/E1/attrs/A1 --payload "$payload" --json -X PUT
```

Comment:
Maybe is due to the mismatch described in a comment above, but this operaiton maybe should return an error. I doesn't make sense to do a PUT on /v2/entities/E1/attrs/A1 without including at least one of the following properties: `value`, `type` or `metadata`.

If the operation where ` PUT /v2/entities/E1/attrs/A1/value`, then it would make sense to have `{ }` as payload. In this case, I would suggest to use "(empty JSON object)" instead of just "(empty)" in the test description, to make it more clear.